### PR TITLE
fixing tiny.cloud urls that have changed

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Fixing some links that moved during product site refresh

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
 enterpriseplan: Tiny Custom Plans
 ## common tiny.cloud URLs
 supporturl: https://support.tiny.cloud
-plugindirectory: https://www.tiny.cloud/apps/
+plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
-accountpageurl: https://apps.tiny.cloud/my-account
+accountpageurl: https://www.tiny.cloud/my-account/
```
Comment (tylerkelly13) [inline _config.yml:40]:
```suggestion
accountpageurl: https://www.tiny.cloud/my-account
```

Comment (tylerkelly13) [reply _config.yml]: just in case the `/` was missing for a reason.

**Review by ltrouton — APPROVED** [review]

**Review by ashfordneil — APPROVED** [review]

File: _config.yml [hunk 2]
```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
 enterpriseplan: Tiny Custom Plans
 ## common tiny.cloud URLs
 supporturl: https://support.tiny.cloud
-plugindirectory: https://www.tiny.cloud/apps/
+plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
-accountpageurl: https://apps.tiny.cloud/my-account
+accountpageurl: https://www.tiny.cloud/my-account
```
Comment (lnewson) [inline _config.yml:40]: This doesn't look correct, as it's immediately redirected to `https://www.tiny.cloud/my-account/` so shouldn't it be that instead?

**Review by lnewson — APPROVED** [review]

File: _config.yml (hunk 2 above)
Comment (tylerkelly13) [reply _config.yml:40]: I'll check where we use this attribute to make sure it wont break anything. typically we use these URLs as `{{site.attribute}}/page/` or `{{site.attribute}}` and let it redirect to the slash version

Comment (lnewson) [reply _config.yml:40]: ahh, right I forgot about that sorry :cry:

Comment (tylerkelly13) [reply _config.yml:40]: no worries. I'll leave as is. (it's easier to read in text as-is)

File: _config.yml [hunk 3]
```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
...
 accountsignup: https://www.tiny.cloud/auth/signup
 productpages: https://apps.tiny.cloud/products
```
Comment (EkimChau) [inline _config.yml:42]: would this one day need to be https://www.tiny.cloud/products/

File: _config.yml (hunk 2 above)
Comment (EkimChau) [inline _config.yml:40]: we should add trailing slashes/ for all the links, it should save on 301 redirects

**Review by EkimChau — APPROVED** [review]

File: _config.yml (hunk 3 above)
Comment (tylerkelly13) [reply _config.yml:42]: hopefully. But for now that is correct.

File: _config.yml (hunk 2 above)
Comment (tylerkelly13) [reply _config.yml:40]: in the current system, baseurl and url are set by the site, such as:
```html
<body role="document" data-shared-baseurl="" data-baseurl="/docs">
```
so if we do this, some will have the trailing slash, others wont; which could get confusing. Hopefully we will eventually fix this.

**Pull request merged**